src/components/Games/Number/NumberMain.js

```diff
@@ -47,28 +60,43 @@ const Input = styled.input`
 `;
 
 export const NumberMain = () => {
-  const [num, setNum] = useState(0);
+  const [num, setNum] = useState(Math.floor(Math.random() *  (9 - 1 +1)) +1);
+  const [newNum, setNewNum] = useState(Math.floor(Math.random() *  (9 - 1 +1)) +1);
```

Comment:
You don't really need this as state, you can generate a new random number on click and add it to the `num` state